__tests__/app.test.js

```diff
@@ -146,4 +146,50 @@ describe("/api/articles", () => {
         });
     });
   });
+  describe("/api/articles", () => {
+    describe("GET", () => {
+      test("GET 200: responds with 200 status", () => {
+        return request(app).get("/api/articles").expect(200);
+      });
+      test("GET 200: responds with array of all articles, length 13", () => {
+        return request(app)
+          .get("/api/articles")
+          .expect(200)
+          .then(({ body }) => {
+            const { articles } = body;
+            expect(articles.length).toBe(13);
+          });
+      });
+      test("GET 200: responds with array of articles each having author, title, article_id, topic, created_at, votes, article_img_url, comment_count properties ", () => {
+        return request(app)
+          .get("/api/articles")
+          .expect(200)
+          .then(({ body }) => {
+            const { articles } = body;
+            expect(articles.length).toBe(13);
+            articles.forEach((article) => {
+              expect(article).toMatchObject({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+                comment_count: expect.any(String),
```

Comment:
Should a comment count be a string? What could you change on your sql query to make this a number?